settings.gradle

```diff
@@ -38,4 +39,4 @@ include 'fabric-rendering-fluids-v1'
 include 'fabric-resource-loader-v0'
 include 'fabric-tag-extensions-v0'
 include 'fabric-textures-v0'
-include 'fabric-dimensions-v1'
```

Comment:
Not sure whats gone on here, only a minor thing.

Reply:
Just beware of merge conflicts.